internal/docker/container.go

```diff
@@ -262,10 +263,27 @@ func (r *ContainerRunner) runJobs(containerOpts <-chan containerStartOptions, re
 		}
 		start := time.Now()
 		containerID, output, jobDetails, passed, skipped, err := r.runSuite(opts)
+
+		browser := opts.Browser
+		// Playwright uses its own bundled browsers, not the ones installed in the image
+		if r.Framework.Name != "playwright" {
+			if browser == "firefox" {
+				version, err := r.docker.FirefoxVersion(r.Ctx, opts.Docker.Image)
```

Comment:
just curious about error handling here. any specific thoughts to abandon the error here?

Reply:
I'm not sure what to do about an error here other than ignore it. Open to suggestions!